### PR TITLE
Tas 26 & Tas 27 create a add  at the bottom of the dashboard and implement design ticket

### PR DESCRIPTION
**Pull request by Agata-Duda**
- Implemented Add button on Dashboard page
- Implemented redirect upon signing up as a new user from the get started fragment
- Implemented redirect from add button to add task fragment
- Implemented Add Task fragment 
- Updated Nav.xml files
- Tested Creating Tasks with physical device and checked populating of firestore
- Handled empty required fields
- On success redirect to dashboard fragment 

Also including Tas 27
Implemented design for ticket to be displayed

### CODE REVIEW

File: app/src/main/res/navigation/nav.xml [hunk 1]
```diff
@@ -57,8 +57,8 @@
         android:label="fragment_get_started"
         tools:layout="@layout/fragment_get_started" >
         <action
-            android:id="@+id/action_getStartedFragment_to_dashboard"
-            app:destination="@id/dashboard" />
+            android:id="@+id/action_getStartedFragment_to_addTask"
+            app:destination="@id/addTask" />
```
Comment (psavarmattas) [inline app/src/main/res/navigation/nav.xml:61]: Why is the user going from Get Started Fragment to add task. As this is an internal application the task should already be there even if he/she is a new user.

**Review by psavarmattas — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/taskmuse/app/ui/fragment/task/AddTaskFragment.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        assigneeAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+        taskAssigneeSpinner.setAdapter(assigneeAdapter);
+
+        // Set OnClickListener for the "Add Task" button
+        Button addTaskButton = view.findViewById(R.id.addTaskButton);
+        addTaskButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                // Retrieve values from UI elements
+                String id = "";
```
Comment (psavarmattas) [inline app/src/main/java/com/taskmuse/app/ui/fragment/task/AddTaskFragment.java:84]: This is not required as it is creating a field which was not discussed as the format.

**Review by psavarmattas — CHANGES_REQUESTED** [review]

File: app/src/main/res/navigation/nav.xml (hunk 1 above)
Comment (Agata-Duda) [reply app/src/main/res/navigation/nav.xml:61]: We discussed this previously, the user should be directed to the add task page so they can create their first task easily

File: app/src/main/java/com/taskmuse/app/ui/fragment/task/AddTaskFragment.java (hunk 2 above)
Comment (psavarmattas) [reply app/src/main/java/com/taskmuse/app/ui/fragment/task/AddTaskFragment.java:83]: I don't think we are using the ID anywhere specifically can we remove it from the Task.java file?

Comment (Agata-Duda) [reply app/src/main/java/com/taskmuse/app/ui/fragment/task/AddTaskFragment.java:83]: It's being used in edit task fragment to assign ID from the collection

**Review by kajal1106 — APPROVED** [review]

**Review by psavarmattas — APPROVED** [review]
LGTM. Greate work

File: app/src/main/java/com/taskmuse/app/ui/fragment/dashboard/dashboard.java [hunk 3]
```diff
@@ -65,6 +68,9 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         fetchAndPopulateTasks(STATUS_DONE, recyclerViewDone, "");
 
         searchEditText = view.findViewById(R.id.searchEditText);
+
```
Comment (sandeshkulk) [inline app/src/main/java/com/taskmuse/app/ui/fragment/dashboard/dashboard.java:71]: 3 lines spaces is really needed?

**Review by psavarmattas — CHANGES_REQUESTED** [review]
Please remove the redundant edit icon files.

File: app/src/main/res/drawable/edit_icon_background.xml
Comment (psavarmattas) [inline app/src/main/res/drawable/edit_icon_background.xml:1]: This is also not needed

**Review by psavarmattas — CHANGES_REQUESTED** [review]

**Review by psavarmattas — APPROVED** [review]
LGTM. Great work

**Pull request merged**